src/plugins/fixedHeader.js

```diff
@@ -0,0 +1,19 @@
+const cheerio = module.parent.require('cheerio');
+
+module.exports = {
```

Comment:
don't think we should do this as a plugin since it is a core markbind feature.
You could look to `insertHeaderFile` and `_parse` to integrate this directly

Reply:
I can do that in `postRender` - the same place where plugins are applied. Will there be extra concerns by doing that?

Reply:
mainly of code organization and performance secondly ( as putting it there means you are parsing the entire `content` one more time, keeping the number of parses to a minimum would be nice )

Unless absolutely necessary ( like in the case of id resolution in popovers ), I think each stage of page generation should maintain a single unit of functionality as is currently. `postRender` is for plugins - you could certainly add it immediately outside of `postRender`, but adding one entire stage of parsing just for fixed navbars seems a little bit of waste, and there is already similar parsing going on in `_parse/insertHeaderFile`. Would make sense to put it there from both standpoints

Reply:
If we're to implement this inside `_parse`, fixed header would be a config option in `site.json` I suppose?

Reply:
you could still 'detect' it during `insertHeaderFile` - though just to confirm the current implementation is `<header class="header-fixed">` right? 😮